### PR TITLE
Add Appveyor configuration for Windows build.

### PR DESCRIPTION
**Pull request by antiagainst**
Tested and worked: https://ci.appveyor.com/project/antiagainst/spirv-tools

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Windows Build Configuration for AppVeyor
+# http://www.appveyor.com/docs/appveyor-yml
+
+# version format
+version: "{build}"
+
+os: Visual Studio 2013
```
Comment (dekimir) [inline .appveyor.yml]: If possible, I'd prefer to go with 2015.

**Comment by antiagainst** [comment]
@dneto0 could you please turn on Appveyor for SPIRV-Tools and also change the custom `.yml` file name to `.appveyor.yml`? And the badge link, too. ;)

**Comment by antiagainst** [comment]
@dneto0 oh, nvm. I can do it on my side.

File: .appveyor.yml (hunk 1 above)
Comment (antiagainst) [inline .appveyor.yml]: Done

**Comment by dneto0** [comment]
+2

**Pull request merged**